.github/workflows/tests.yml

```
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12-dev"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12-dev"]
```

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```
Doesn't `"3.12"` work now without the `-dev`?
If you want to keep this PR focussed on removing 3.7 we can handle this separately.

---

Yep, updated!